src/Microsoft.Azure.WebJobs.Host/Timers/TaskSeriesTimer.cs

```diff
@@ -21,7 +21,7 @@ internal sealed class TaskSeriesTimer : ITaskSeriesTimer
         private Task _run;
         private bool _disposed;
 
-        public TaskSeriesTimer(ITaskSeriesCommand command, IBackgroundExceptionDispatcher backgroundExceptionDispatcher,
+        public TaskSeriesTimer(ITaskSeriesCommand command, IWebJobExceptionHandler backgroundExceptionDispatcher,
```

Comment:
Once the interface name is finalized, can you change the parameter names to match throughout all these files?